Source/WebCore/css/StyleRuleImport.h

```diff
@@ -77,7 +79,7 @@ class StyleRuleImport final : public StyleRuleBase {
     void setCSSStyleSheet(const String& href, const URL& baseURL, const String& charset, const CachedCSSStyleSheet*);
     friend class ImportedStyleSheetClient;
 
-    StyleRuleImport(const String& href, MQ::MediaQueryList&&, std::optional<CascadeLayerName>&&);
+    StyleRuleImport(const String& href, MQ::MediaQueryList&&, std::optional<CascadeLayerName>&&, String&& supports, bool supportsSupported);
```

Comment:
Can you pass both these as a struct (like `SupportsCondition`)? You already have `struct Supports` in the caller. Then you need fewer awkward names and no magical booleans.